sunscreen/benches/fractional_range_proof.rs

```diff
@@ -140,5 +158,217 @@ fn fractional_range_proof(_c: &mut Criterion) {
     println!("Verifier time {}s", verifier_time.elapsed().as_secs_f64());
 }
 
-criterion_group!(benches, fractional_range_proof);
+/// Imagine a multi-party computation where each user submits a value between
+/// zero and a maximum encrypted under their key. The MPC computes the mean and
+/// variance of their inputs.
+/// 
+/// This proof is similar to [`private_tx_fractional_range_proof`] except we're only
+/// proving 0 < a <= b where
+/// 
+/// * a is the submitted value under a given user's key.
+/// * b is the maximum value encrypted under the same user's key.
+fn mean_variance_fractional_range_proof(_c: &mut Criterion) {
```

Comment:
This function is so similar to `private_tx_fractional_range_proof`. Potentially parts could be refactored to be shared between the benches.

Reply:
I created a `make_fractional_value` that significantly reduced the amount of boilerplate. There is a certain art to be had in violating DRY in tests so they're easier to follow, so I think we should keep the flow of compile, prove, verify